src/components/cluster/detail/node_pool_dropdown_menu.js

```diff
@@ -0,0 +1,36 @@
+import DropdownMenu from '../../UI/dropdown_menu';
```

Comment:
Let's use the absolute path now that we can? Also in some other places.

Reply:
🤦‍♂true